notebooks/00020_fix_positions_hg38_missense_pathogenic.ipynb

```diff
@@ -0,0 +1,2090 @@
+{
```

Comment:
<p>agree</p><p>in position X:154413499 (hg38) is A</p><p>c.302A&gt;T</p><p>p.Asp101Val</p><p>ACMG VOUS</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/zdenkas/human-tafazzin-variants-database/pull/15/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>